frontend/src/components/Yippey/Yippey.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import gift from '../../assets/gift.svg';
+
+import './Yippey.scss'
+
+const Yippey = ({ text, container }) => (
+  <div className={`Yippey Yippey--${container ? 'container': ''}`}>
```

Comment:
When would we not need the container?

Reply:
We don't container props When we use Yippey as sub-component as it's been use in the `OrderForm`